javascript/javascript2/week1/spirit_animal/name_navigator.js

```diff
@@ -0,0 +1,56 @@
+// Create an input element
+const inputField = document.createElement('input');
+inputField.setAttribute('type','text');
+inputField.setAttribute('placeholder','Enter your name');
+inputField.setAttribute('class','name_input');
+const spiritAnimalNavigator = document.getElementById('spirit-animal-navigator');
+spiritAnimalNavigator.appendChild(inputField);
+
+// Create a button
+const myButton = document.createElement('button');
+myButton.setAttribute('class','button');
+myButton.innerText = 'Let`s go!';
+spiritAnimalNavigator.appendChild(myButton);
+
+// Create a tag to display the spirit animal info
+const info = document.createElement('p');
+spiritAnimalNavigator.appendChild(info);
```

Comment:
I would write all the static HTML in the `.html` file, only dynamically added HTML with the Javascript DOM API